### PR TITLE
Implement "init" command

### LINKED ISSUE #41: `lorri init`
Set up a project to work with lorri and direnv.
 
For now that means:

* Make sure `shell.nix` exists and is a valid shell file
* `.envrc`

If the files don’t exist, we simply create them and populate them with our setup code. If they exist, we don’t touch them and print an informational message, like:

* “`shell.nix` exists, skipping. Make sure it is of a form that works with `nix-shell`.”
* “`.envrc` exists, skipping. Please add eval `"$(lorri direnv)"` to it to set up lorri support.”

In the case of `shell.nix` we could also try to find out whether it is a correct shell file.

### PR DESCRIPTION
**Pull request by gilligan**
Implements an `init` command that writes a default shell.nix and .envrc to $CWD if those files don't exist already.

Should fix #41 (?) however it does nothing about validating whether or not the existing `shell.nix` is valid - which is something @Profpatsch mentioned in #41

### CODE REVIEW

File: src/ops/init/mod.rs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    }
+}
+
+/// See the documentation for lorri::cli::Command::Init for
+/// more details
+pub fn main(default_shell: &str, default_envrc: &str) -> OpResult {
+    println!("Bootstrapping a new lorri project\n");
+
+    to_op(create_if_missing(Path::new("./shell.nix"), default_shell))?;
+    to_op(create_if_missing(Path::new("./.envrc"), default_envrc))?;
```
Comment (Profpatsch) [inline src/ops/init/mod.rs]: I’d add the better “exists” messages I listed in #41 here:

* “shell.nix exists, skipping. Make sure it is of a form that works with nix-shell.”
* “.envrc exists, skipping. Please add eval "$(lorri direnv)" to it to set up lorri support.”

File: src/ops/init/mod.rs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    Ok(Some(String::from("\nSetup done.")))
```
Comment (Profpatsch) [inline src/ops/init/mod.rs]: nit: `ops::ok_msg()`

File: src/ops/init/mod.rs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    match e {
+        Ok(_) => ok(),
+        Err(e) => ExitError::errmsg(format!("{}", e)),
+    }
+}
+
+/// See the documentation for lorri::cli::Command::Init for
+/// more details
+pub fn main(default_shell: &str, default_envrc: &str) -> OpResult {
+    println!("Bootstrapping a new lorri project\n");
```
Comment (Profpatsch) [inline src/ops/init/mod.rs]: I think we can leave out this message, because the user ostensibly requests the bootstrap when she types `lorri init`.

Comment (Profpatsch) [reply src/ops/init/mod.rs]: Or, or leave it in, but use it to print the directory: `"… in /home/user/current/working/dir"`.

File: src/ops/init/mod.rs (hunk 2 above)
Comment (gilligan) [reply src/ops/init/mod.rs]: Okay cool, will do

File: src/ops/init/mod.rs (hunk 3 above)
Comment (gilligan) [reply src/ops/init/mod.rs]: Actually I think you were right - I agree that I should just remove it

File: src/ops/init/mod.rs (hunk 1 above)
Comment (gilligan) [reply src/ops/init/mod.rs]: A bit wordy for my taste but on the other hand i don't have strong feelings about that ;-)
